types/types.d.ts

```diff
@@ -27,7 +27,7 @@ interface SessionData extends ExpressSessionData {
   touch(): void;
 
   /** Regenerates the session by generating a new `sessionId`. */
-  regenerate(): void;
+  regenerate(request: Fastify.FastifyRequest): void;
```

Comment:
Should I change to a nullable or keep it like this?

Reply:
cc @fastify/typescript

Reply:
Ok, let's make it optional, so we don't break anything.
But make sure that it is clearly documented.